openstack/cinder/Chart.yaml

```diff
@@ -6,25 +6,25 @@ version: 0.2.1
 dependencies:
   - name: utils
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: ~0.18.4
+    version: ~0.22.1
   - name: mariadb
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: 0.14.2
+    version: 0.15.4
```

Comment:
meanwhile we have a [0.15.5](https://github.com/sapcc/helm-charts/blob/master/common/mariadb/CHANGELOG.md)